protos/dcs/coalition/coalition.proto

```diff
@@ -23,6 +23,100 @@ service CoalitionService {
    */
   rpc GetMainReferencePoint(GetMainReferencePointRequest)
       returns (GetMainReferencePointResponse) {}
+
+  // https://wiki.hoggitworld.com/view/DCS_func_getPlayers
+  rpc GetPlayers(GetPlayersRequest) returns (GetPlayersResponse) {}
+}
+
+message AddGroupRequest {
+	dcs.common.Country country = 1;
+	dcs.common.GroupCategory category = 2;
+	oneof template {
+	  GroundGroupTemplate ground_template = 3;
+	  ShipGroupTemplate ship_template = 4;
+	  HelicopterGroupTemplate helicopter_template = 5;
+	  PlaneGroupTemplate plane_template = 6;
+	}
+
+	message GroundGroupTemplate {
+		optional uint32 group_id = 1;
+		optional bool hidden = 2;
+		optional bool late_activation = 3;
+		string name = 4;
+		dcs.common.Position position = 5;
+		repeated Point route = 6;
+		optional uint32 start_time = 7;
+		string task = 8;
+		bool task_selected = 9;
+		repeated Task tasks = 10;
+		bool uncontrollable = 11;
+		repeated GroundUnitTemplate units = 12;
+		optional bool visible = 13;
```

Comment:
We also have `hidden` above - do they have different effects?

Reply:
Of course they are different effects! doesn't the naming make that _perfectly clear?!_ 😁